docs/learn/learn-auction.md

```diff
@@ -125,7 +149,7 @@ have ended before they even bid.
 
 ## Polkadot Implementation
 
-Polkadot will use a _random beacon_ based on the VRF that's used also in other places of the
+Polkadot will use a random beacon_ based on the VRF that's used also in other places of the
```

Comment:
Was the intention to remove both the underscores?

Reply:
accidentally removed it seems